docs/instances.rst

```diff
@@ -40,17 +40,32 @@ Local Instances
 To add local instances to the ``instances`` key, we add a list of dictionaries with two keys
```

Comment:
two keys and an optional third one

Reply:
I've fixed the docs entry.

There's also a new test to check the functionality of the `instdir` key and the instance `subdir` keys. The instance files are just empty files and the test checks if all instances are available.